tools/pipelines/build-eslint-plugin-fluid.yml

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# build-eslint-plugin-fluid pipeline
+
+name: $(Build.BuildId)
+
+parameters:
+- name: releaseBuildOverride
+  displayName: Release Build (default = not released)
+  type: string
+  default: none
+  values:
+    - none
+    - prerelease
+    - release
+- name: publishOverride
+  displayName: Publish Override (default = based on branch)
+  type: string
+  default: default
+  values:
+    - default
+    - skip
+    - force
+- name: buildToolsVersionToInstall
+  displayName: Fluid build tools version (default = installs version in repo)
+  type: string
+  default: repo
+
+trigger:
+  branches:
+    include:
+    - main
+    - next
+    - lts
+    - release/*
+  paths:
+    include:
+    - .prettierignore
+    - common/build/eslint-plugin-fluid
```

Comment:
We'll need to add `tools/markdown-magic` here as one of the trigger paths as well, since we use a `file:...` dependency on it. Otherwise this package's build won't re-run when changes are made to markdown-magic, even though they should.

It would probably be a good idea to add that to `build-eslint-config-fluid.yml` as well, while we're here.

Reply:
We should annotate it with a comment explaining that we need it because of the `file` dev dependency.
See here for an example: https://github.com/microsoft/FluidFramework/blob/main/tools/pipelines/build-api-markdown-documenter.yml#L35

Reply:
Added 😄.